### ISSUE TITLE
Restore fuctionality to set exact limit output of product or ingredient

### ISSUE THREAD

**Issue by RobynTheDevil** [ClaudeMetz/FactoryPlanner#418]
### Suggestion

Previous versions allowed you to right click an ingredient or product and directly set the desired output/input amount. The functionality appears removed. You can achieve this with the percentage output but requires repetitive calculation and inaccurate tuning.

**Comment by ClaudeMetz** [comment]
Thanks for the suggestion. I am planning on adding this functionality back, with a more solid technical foundation.

**Comment by RobynTheDevil** [comment]
Oh nice didnt see a mention so i thought maybe it got shelved thank you

**Comment by ClaudeMetz** [comment]
Yeah I thought I had a card for it too, but apparently not, so now your card is the blessed one :p

**Comment by ClaudeMetz** [comment]
This logic or UI could potentially also be used by the item picker.

**Comment by ClaudeMetz** [comment]
This might not get done after all, as the mechanism it uses of setting line percentages is not compatible with simplex or gaussian solvers currently. It could potentially be made to work, but what's more likely is that the percentage feature will be dropped alltogether. It is just not as required anymore with the new and improved solvers, plus the input-based solving feature obviates some need for it.

There is the possibility that a different version of this can be done, where it edits top level item amounts instead of setting line percentages. That might be tricky to do, and have some annoyances, but it should be possible with all solvers. We'll see where it goes after solver improvements are through.
